saleor/graphql/schema.graphql

```diff
@@ -2316,10 +2316,10 @@ type Mutation {
   orderUpdateShipping(order: ID!, input: OrderUpdateShippingInput): OrderUpdateShipping
   orderVoid(id: ID!): OrderVoid
   orderBulkCancel(ids: [ID]!, restock: Boolean!): OrderBulkCancel
-  deleteMetadata(id: ID!, key: String!): DeleteMetadata
+  deleteMetadata(id: ID!, keys: [String!]!): DeleteMetadata
   deletePrivateMetadata(id: ID!, key: String!): DeletePrivateMetadata
```

Comment:
The schema is outdated? Here we should also have `keys`.

Reply:
Why Travis doesn't build this PR?

Reply:
in delete mutations there was `key` I branch out from updated master branch so no idea

Reply:
@koradon In Saleor we use pre-commit with some checks. For example check for actual graphql schema. Maby you should install it because these checks get a result faster than Travis.

Reply:
I think Travis didn't start because of merge conflicts. I've fixed them and now it's failing with outdated schema - it needs to be generated again.

Reply:
ow now i see it, looks like I forgot one `s`

Reply:
Done